build/networking/README_DOCKER.md

```diff
@@ -130,7 +130,7 @@ configuration file settings and inner workings!
     mkdir -p "${VOLUME}"/intf
   * docker run --name "${CONTAINER_NAME}" --cap-add ALL --privileged \
                -v "${VOLUME}":/tmp -p 9339:9339 -p 9559:9559 \
-               -d --entrypoint /root/scripts/start.sh -it <Image ID> rundaemon
+               -d --entrypoint /root/scripts/start.sh -it < Image ID > rundaemon
 
 * Run `ipdk connect` - To connect to your IPDK container daemon and to use
```

Comment:
"Run `ipdk connect` to connect to your IPDK container..." (no hyphen after `ipdk connect`)

Reply:
Fixed